app/models/variable_draft.rb

```diff
@@ -8,6 +8,28 @@ class << self
     def forms
       []
     end
+
+    def create_from_variable(variable, user, native_id)
+      new_long_name = (variable['LongName'].blank? ? nil : variable['LongName'])
+
+      if native_id
+        # Editing a record
+        draft = VariableDraft.find_or_initialize_by(native_id: native_id)
+        draft.entry_title = new_long_name
+        draft.short_name = (variable['Name'].blank? ? nil : variable['Name'])
+      else
```

Comment:
I wasn't sure what to do with the case where a native_id didn't come through properly, so I left in what was happening with collections and cloning. Should I just remove this, make it save a new native_id, or something else?